src/Lykke.Service.ClientAssetRule/Startup.cs

```diff
@@ -2,6 +2,7 @@
 using System.Threading.Tasks;
```

Comment:
please add changes related to ENV_INFO (https://github.com/LykkeCity/lykke.dotnettemplates/commit/bc36f013dc7ab6dc44cd5e772de61cf0ef1de272)

Reply:
done